<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QDialog" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>302</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UPDATE ZCP015</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 245, 245);
width:580;
height:300;

QDialog:{
	padding:5px;
}
button:{
	border:none;
	border-radius:5px
}</string>
  </property>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>241</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:'white';

border:1px solid #0072ff;
color:#0072ff;
font-weight:600;
box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
border-radius:5px;

</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run_process_btn">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>241</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 114, 255);
border:none;
color:'white';
font-weight:600;
box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Processar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_file_btn">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>106</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 114, 255);
border:none;
color:'white';
font-weight:600;
box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
border-radius:5px;

cursor:pointer;</string>
   </property>
   <property name="text">
    <string>Procurar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_path">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>107</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #0072ff;
border-radius:5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>6</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(84, 84, 84);</string>
   </property>
   <property name="text">
    <string>Atualização de Base de Dados ZCP015</string>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>70</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 114, 255);
font-size:18px;</string>
   </property>
   <property name="text">
    <string>Selecione a Base com os novos dados</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>153</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:{
	20px
}</string>
   </property>
   <property name="text">
    <string>Exportar Base do SAP</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Atualizar Base ZCP015</string>
   </property>
  </widget>
  <widget class="QLabel" name="process">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>189</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 114, 255);
font-size:18px;</string>
   </property>
   <property name="text">
    <string>Processando...</string>
   </property>
  </widget>
  <zorder>exit_button</zorder>
  <zorder>run_process_btn</zorder>
  <zorder>search_file_btn</zorder>
  <zorder>input_path</zorder>
  <zorder>info</zorder>
  <zorder>title</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>process</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
